README.md

```diff
@@ -215,7 +215,7 @@ app.use exprestive
   * default: `__dirname`
 * `controllersPattern`
   * [Glob](https://github.com/isaacs/node-glob) pattern used to find controllers. Resolved relative to `appDir`
-  * default: `'controllers/*_controller.{coffee,js}'`
+  * default: `'controllers/*_controller.{coffee,js,ls}'`
```

Comment:
Agreed, and we don't do it or enforce it. We just are robust enough to handle with this smelly code. Be generous in what you accept... Alternatively omit the "c" altogether, as I have suggested.